### DISCUSSION TITLE
Re-Routing to All Pins "Page"/ Showing All Pins When Deleting/Updating Pins From Board Specific View

### DISCUSSION THREAD

**Discussion by thedigitalmenagerie** [nss-evening-cohort-14/evening-client-side#183] [Help Me!] [answered]
Honey-Rae Swan 

Have you searched the other help tickets? 
Yep!

Describe your issue
Everything was working and I pushed to main but after, when in a specific board view deleting or updating pins, I tried the deleting and updating. It is executed but I am either "re-routed" to the all pins "page" or they are all popping up in board view. 

List any error messages that appear
I don't have any error or warning messages. 

What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
I am not really sure sure what the issue is or what to google here to solve it and am making this issue ticket so that I might be able to learn what to google. 

What have you tried? Tell us everything you have tried.
I tried having different functions for showPins (this already existed and is still being used), update, and delete that were specific to board specific view.  

Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with NO SCREEN SHOTS 🥰
https://github.com/thedigitalmenagerie/ASSIGNMENT-Firebase-Pinterest

**Comment by drteresavasquez** [comment]
Check your click event on your submit button.

**Comment by drteresavasquez** [comment, marked as answer]
@thedigitalmenagerie looks like you got this resolved? Can you update this ticket?
